backend/src/user/entities/user.entity.ts

```diff
@@ -18,13 +27,42 @@ export class User {
 	@Column({ unique: true, nullable: false })
 	name: string;
 
-	@Column({ nullable: false })
 	@Exclude()
+	@Column({ nullable: false })
 	password: string;
 
 	@CreateDateColumn()
 	created_at: Date;
 
 	@UpdateDateColumn()
 	updated_at: Date;
+
+	@IsOptional()
+	@OneToOne(
+		() => MatchmakingRequest,
+		(matchRequest: MatchmakingRequest) => matchRequest.user,
+	)
+	matchmaking_request: MatchmakingRequest;
+
+	@Exclude()
+	@IsOptional()
+	@OneToMany(() => Game, (game: Game) => game.player_one)
+	games_as_player_one: Game[];
+	
+	@Exclude()
+	@IsOptional()
+	@OneToMany(() => Game, (game: Game) => game.player_two)
+	games_as_player_two: Game[];
+	
+	@IsOptional()
+	@Expose()
+	get games(): Game[] {
+		//  if games_asPlayerone/two are null set them = []
+		return [...(this.games_as_player_one ?? []), ...(this.games_as_player_two ?? [])];
+	}
+
+	@IsOptional()
+	@ManyToOne(() => GameInvite, (invite) => invite.players)
+	invite: GameInvite;
+
```

Comment:
not sure if you need this column, since both players exist in the other table? 
I think i wrote it somewhere else but I cant find it anymore, if so please ignore!